torch/csrc/api/include/torch/special.h

```diff
@@ -182,6 +182,22 @@ inline Tensor& xlog1py_out(Tensor& result, const Tensor& self, const Scalar& oth
   return torch::special_xlog1py_out(result, self, other);
 }
 
+/// Computes the zeroth order modified Bessel function of the first kind of input, elementwise
+/// See https://pytorch.org/docs/master/special.html#torch.special.i0
+///
+/// Example:
+/// ```
+/// auto t = torch::randn(128, dtype=kDouble);
+/// torch::special::i0(t);
+/// ```
+inline Tensor i0(const Tensor& self) {
+  return torch::special_i0(self);
+}
+
+inline Tensor i0_out(const Tensor& self) {
```

Comment:
nit: `inline Tensor i0_out(const Tensor& self)` -> `inline Tensor& i0_out(Tensor& result, const Tensor& self)`

Reply:
Good catch, @kshitij12345; I always miss that ampersand

Reply:
Thanks, @kshitij12345 - looks like ops like `ndtr, i0e, i1, i1e` are also missing the ampersand (`inline Tensor&`). Is it fine if I modify those in this PR? @mruberry @kshitij12345

Reply:
Fixing the out variants sounds good to me

Reply:
Ouch. Thanks! That will be great.

Reply:
Thanks, should be resolved in the latest commit. :)